rts/Rendering/Map/InfoTexture/Modern/Height.cpp

```diff
@@ -44,7 +44,7 @@ CHeightTexture::CHeightTexture()
 			.reqNumLevels = 1,
 			.minFilter = GL_NEAREST,
 			.magFilter = GL_LINEAR,
-			.wrapModes = std::initializer_list{ GL_REPEAT, GL_CLAMP_TO_EDGE }
+			.wrapModes = std::array<int32_t, 3>{ GL_REPEAT, GL_CLAMP_TO_EDGE, GL_REPEAT } // R wrap unused for 2D textures
```

Comment:
Where did the 3rd element and its value come from?

Reply:
The array has to be of a fixed size, in this case of 3, just to cover a rare case of 3D textures.